basic-conditionals.tex

```diff
@@ -0,0 +1,400 @@
+    \PY{k+kt}{int} \PY{n}{zelja}\PY{p}{;}
+    \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Napisi 1, ce zelis juho, in 2, ce zelis sendvic.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+
+    \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Kaksno juho zelis? Napisi 1 za govejo in 2 za paradiznikovo.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+        \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Ena goveja juha. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Ena paradiznikova juha. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}}
+    \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Kaksen sendvic zelis? Napisi 1 za sunko, in 2 za vegi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{zelja}\PY{p}{)}\PY{p}{;}
+        \PY{k}{if} \PY{p}{(}\PY{n}{zelja} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{1}\PY{p}{)} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{En sendvic s sunko. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+            \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{En vegeterjanski sendvic. Dober tek!}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+        \PY{p}{\PYZcb{}}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+Napisi 1, ce zelis juho, in 2, ce zelis sendvic.\\
+2\\
+Kaksen sendvic zelis? Napisi 1 za sunko, in 2 za vegi.\\
+2\\
+En vegeterjanski sendvic. Dober tek!
+\end{inout}
+\end{examples}
+
+\pagebreak
+Pozorni bodimo tudi na postavitev kode. Običajno kodo znotraj zavitih oklepajev \verb+if+ stavka pišemo tako, da je poravnana za štiri presledke desno od kode zunaj \verb+if+ stavka. V nekaterih programskih jezikih je taka poravnava obvezna, v C/C++ pa ne, vendar nezamaknjena koda že v zelo majhnih programih postane popolnoma nepregledna. Branje in popravljanje kode je veliko lažje, če del kode znotraj zavitih oklepajev zamaknemo za štiri presledke. Za to lahko uporabimo tudi tipko Tab, ki se na tipkovnici nahaja levo od tipke Q.
```

Comment:
Poleg tega bi jaz tale odstavek cel dal v razdelek pogoste napake.

Reply:
Sem dal, ampak se mi ta sprememba ne zdi dobra. Ko jaz berem take stvari, so mi okenca kratka in jedrnata opomba na nekaj specifičnega. Če je okenc preveč, ali če so predolga, potem izgubijo svojo moč. Dolge razlage, ki so aplikativne povsod, naj bodo v glavnem tekstu; sploh, če se bomo kasneje nanašali nanje (npr. v loopih, "... zamik mora biti tak kot v if ..."

Reply:
Meni se zdi, da vsebinsko to gre pod errorje, je pa res malo dolgo. Ce bi se skrajsalo gre lahko v error.

Reply:
Kako pa misliš skrajšati?